tir/authorizationClient.go

```diff
@@ -35,13 +37,34 @@ type NoAuthHttpClient struct {
 	httpClient HttpClient
 }
 
+func (ac AuthorizingHttpClient) FillMetadataCache(context.Context) {
+	tirEndpointsInterface, hit := common.GlobalCache.TirEndpoints.Get("tirEndpoints")
+	if !hit {
+		logging.Log().Info("issuers list not found in cache")
+		return
+	}
+
+	tirEndpoints := tirEndpointsInterface.([]string)
+	for _, tirEndpoint := range tirEndpoints {
+		metaData, err := ac.getMetaData(tirEndpoint)
+		if err != nil {
+			logging.Log().Errorf("Was not able to get the openid metadata. Err: %v", err)
+			return
```

Comment:
Having one failing endpoint would stop the update of all other endpoints if we return here right?